GliaWidgets/SecureConversations/ChatTranscript/SecureConversations.TranscriptModel.swift

```diff
@@ -589,18 +591,22 @@ extension SecureConversations.TranscriptModel {
             // We no longer need to listen to Interactor events,
             // so unsubscribe.
             self.environment.interactor.removeObserver(self)
-            markMessagesAsRead()
             delegate?(.upgradeToChatEngagement(self))
         case let .receivedMessage(message):
             receiveMessage(from: .socket(message))
+            markMessagesAsRead(delayed: false)
```

Comment:
@EgorovEI here are the changes for received message

Reply:
Oh, sorry. Missed it